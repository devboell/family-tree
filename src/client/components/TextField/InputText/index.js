import styled from 'styled-components'


const InputText = styled.input`
  height: 20px;
  font-size: 14px;
  border: none;
  border-bottom: solid 2px #c9c9c9;
  color: white;
  background-color: rgb(64,64,64);
  &:focus {
     outline: none;
   }
`

export default InputText
