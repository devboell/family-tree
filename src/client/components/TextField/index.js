import React from 'react'
import { PropTypes } from 'prop-types'
import InputText from './InputText'

const TextField = ({ input }) => (
  <InputText
    {...input}
    autoComplete="off"
    type="text"
  />
)

TextField.propTypes = {
  input: PropTypes.shape({}).isRequired,
}


export default TextField
