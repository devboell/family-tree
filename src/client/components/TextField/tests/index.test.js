import React from 'react'
import TextField from '../index'

const props = {
  input: {
    value: 'John',
    onChange: jest.fn(),
  },
}

it('TextField, renders component', () => {
  const wrapper = shallow(<TextField {...props} />)
  expect(wrapper).toMatchSnapshot()
})
