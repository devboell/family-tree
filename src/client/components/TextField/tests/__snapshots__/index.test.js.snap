// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextField, renders component 1`] = `
<styled.input
  autoComplete="off"
  onChange={[Function]}
  type="text"
  value="John"
/>
`;
